LIPS/lip-28.md

```diff
@@ -0,0 +1,106 @@
+2. **Emergency Activation** and **Emergency Execution** Committees are covering the Dual Governace Protected deployment mode: during the limited time after the deployment (proposed 1 year: `max_emergency_protection_duration` in [full parameters list](https://research.lido.fi/t/dual-governance-analytics-note-on-parameters-values/9905#p-21022-appendix-a-26)) the Emergency Activation Committee would be able to trigger emergency mode (blocking the permissionless execution of any motions in Dual Governance pipeline), which in turn allows the Emergency Execution Committee to either 1) execute proposals in the Dual Governance pipeline; or 2) perform an Emergency Reset, effectively disabling the Dual Governance; in practice that action would return the execution rights back to Lido DAO Voting contract [[SPEC](https://github.com/lidofinance/dual-governance/blob/develop/docs/specification.md#protected-deployment-mode)]. The mechanics acts as the mitigation of edge-case vulnerability in the codebase. Both committees are proposed to have the same set of signers with different quorum settings for both Emergency Activation (4/7) and Emergency Execution (5/7) committees. Activation should be able to act reasonably fast (about a day of reaction time is expected), and Execution may have a longer timeframe for taking action.
+3. **Tiebreaker Committee** is designed to overcome potential tie between the protocol and governance in the particular edge-case [[SPEC](https://github.com/lidofinance/dual-governance/blob/develop/docs/mechanism.md#tiebreaker-committee)]. In order for the Tiebreaker to be activated, the tie should be detected onchain first: 1) Dual Governance is in RageQuit state and the Lido on Ethereum protocol withdrawals are paused for a `TiebreakerActivationTimeout` duration; OR 2) the last time DG had been in Normal or Veto Cooldown state more than `TiebreakerActivationTimeout` time ago.
+    
+  Once activated, Tiebreaker can do two things:
+    
+ 1. Execute the blocked proposal in Dual Governance pipeline
+ 2. Unpause any of the paused protocol contracts
+  
+  Tiebreaker is proposed to be set up as “multisig of multisigs”, with 2/3 quorum on top multisig, and independent quorum settings for underlying multisigs.
```

Comment:
would be great to provide numbers and msig structure here, bc the msig has quite a significant power under the proposed setup structure

Reply:
Trying to settle, would be public, don't feel like I could put it into lip now =(